osv/malicious/npm/@evg-ui/lib/MAL-0000-evg-ui-lib.json

```diff
@@ -0,0 +1,41 @@
+{
+  "modified": "2025-06-17T05:39:07Z",
+  "published": "2025-06-17T05:39:07Z",
+  "schema_version": "1.6.8",
+  "id": "",
+  "summary": "Malicious code in @evg-ui/lib package (npm)",
+  "details": "Malicious package. Executes hidden script during install to exfiltrate local IP, hostname, and homedir to an OAST server.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "npm",
+        "name": "@evg-ui/lib"
+      },
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0.0.0"
```

Comment:
One more thing I've noticed is that these should just be `"introduced": "0"`.

See https://ossf.github.io/osv-schema/#special-values.